### ISSUE TITLE
Not working on GitHub Enterprise Server 3.13.3

### ISSUE THREAD

**Issue by colindean** [davidfou/conventionalcomments-web-extension#21]
**Describe the bug**

Recently, the GHES version I use at work went from GHES 3.11 to 3.13. The extension no longer produces the expected dropdowns when commenting on PRs.

**To Reproduce**

1. Open a PR page.
2. Click next to the line of code to show the suggestion box.
3. Observe no convcomm dropdowns.

**Expected behavior**

Observe no convcomm dropdowns.

**Screenshots**

**Environment**
Information to reproduce the bug under the same conditions.

- OS: macOS 14.6.1
- Browser: Firefox 129.0.2
- Url: Internal GHES at git.mycompany.com

**Additional context**

It was working earlier this week before the upgrade.

**Comment by colindean** [comment]
Unsure. Is there a debug mode I can enable?

All I'm seeing in the Console is

```
MouseEvent.mozInputSource is deprecated. Use PointerEvent.pointerType instead.
```

when I click to open a comment. Tracing the source, I _think_ it's coming from CC.

Reloading the page, I see a 

```
TypeError: "focus" is read-only
```

coming from the same script (contentscript.js).

I'm on 

```
Version
2.2.0
Last Updated
August 29, 2024
```

btw

**Comment by davidfou** [comment]
I don't think that's related. It's possible to debug the application and most probably it comes from a css selector not working on this version of GitHub https://github.com/davidfou/conventionalcomments-web-extension/blob/main/src/contentScript/commentEditorExtractors/gitlabCommentEditorExtractorV2.ts.

The type error is definitely something weird, but that's too light to understand what's going on.

Without an access, I can't help you. If you're interesting into helping me on that one, I can propose you pair-session so I wouldn't access the repo and I would guide you with the debugging. Could that be an option?

**Comment by colindean** [comment]
Yeah, down to help! I think my next open time is in about 2 weeks since I've got some travel next week unless you're interested in chatting at ~10:00 pm CT when I'm bored in my hotel.

Digging a little bit, I found that I think you meant the [githubCommentEditorExtractorV2.ts](https://github.com/davidfou/conventionalcomments-web-extension/blob/main/src/contentScript/commentEditorExtractors/githubCommentEditorExtractorV2.ts) since I'm talking about GitHub here.

When I try to use Firefox's debugger on that file, I see 

```
Error while fetching an original source: NetworkError when attempting to fetch resource.
Source URL: moz-extension://704b22f4-4981-4dea-926e-b4f3bc790d1d/src/contentScript/commentEditorExtractors/githubCommentEditorExtractorV2.ts
```

in the pane that I expect to show its source code.

**Comment by davidfou** [comment]
Sorry for the late reply, I was also traveling and forgot about that issue.

Can you check your LinkedIn (that's the only place where I managed to send you a message 😝)? I suck with timezones and best would be to book some time via Calendly.

**Comment by davidfou** [comment]
Thanks again @colindean for your help! Meeting someone using the extension regularly and happy about it boosted my motivation to continue working on the extension!

The code is now merged and deployed. I created the branch [`main-mv2`](https://github.com/davidfou/conventionalcomments-web-extension/tree/main-mv2) which contains the same set of features but without moving to manifest v3. Thanks to that, I'm able to push updates to Firefox as well. Let's hope that soon I'll be able to maintain only a single branch.

It's now only a matter of days until the version `v2.3.3` will be available on Chrome and Firefox.
